docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.3'
+
+services:
+  wordpress:
+    image: wordpress:latest
+    ports:
+      - '8080:80'
+    environment:
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
```

Comment:
What are securing?

Reply:
Logic of password naming really and that's about it, since this is a repo for a plugin, database passwords are not a security threat. However, the choice of the password should be random and should not bleed into other passwords with actual security threats.

Reply:
I'm not really worried about this bleeding into anywhere else, its just a development environment.

Reply:
Yeah the actual password would be shared amongst devs working on the project then set on .env, but if it is really just a dev environment with no security threat, then I think we can leave it at that.